inference-engine/tests/functional/plugin/cpu/single_layer_tests/experimental_detectron_roifeatureextractor.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
since we don't have tests for this layer in SLT, maybe better to move it in SLT?

Reply:
please use `EltwiseLayerTest` as reference
place class class test  declaration here: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/eltwise.hpp
test class  definition here: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/shared_test_classes/src/single_layer/eltwise.cpp
TEST_P here: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/shared/include/single_layer_tests/eltwise.hpp
and instance here: openvino/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests